packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -1,7 +1,35 @@
 import { type TemplateResult } from 'lit';
 import { type ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
 
-import defaultLocale from '../locales/en-gb.json' assert { type: 'json' };
+export enum Language {
```

Comment:
as a general rule of thumb we try to avoid using Enums in our TS. This is partly due to historic performance implications with them, and also the JS they compile to. Instead, we try to favour using native JS features for constants. 

In this instance, I wonder if we could utilise a `Map` or `Set` (if the key/values are unique). We can use the locale as a key and the tenant as a value.

If you're open to it, myself or @xander-marjoram (or both) would be happy to pair with you on Tuesday to refactor this?

Reply:
Do you have an example I can view?